tests/test_testing.py

```diff
@@ -201,6 +208,21 @@ def test_empty_post_uses_default_boolean_value(self):
         assert response.status_code == 200
         assert response.data == {"flag": True}
 
+    def test_post_encodes_data_based_on_json_content_type(self):
+        data = {'data': True}
+        response = self.client.post(
+            '/post-json-view/',
+            data=data,
+            content_type='application/json'
+        )
+
+        if django.VERSION < (2, 1):
```

Comment:
as drf now support django 2.2+ only, can we update the version check and use relevant parts only as code?

Reply:
I must have missed this comment :zipper_mouth_face: